crypto/x509/x509_acert.c

```diff
@@ -0,0 +1,307 @@
+/*
+ * Copyright 2021 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <stdio.h>
+#include <string.h>
+#include <crypto/ctype.h>
+#include <openssl/asn1t.h>
+#include <openssl/x509.h>
+#include <openssl/x509v3.h>
+#include "x509_acert.h"
+
+ASN1_SEQUENCE(OBJECT_DIGEST_INFO) = {
```

Comment:
May be a copy of ASN.1 structure definitions from RFC could be a good comment ?

Reply:
The structure definition for OBJECT_DIGEST_INFO, together with the other structure defined in this file
gets to be fairly long.  I've added a comment referencing the RFC and section number as is done in other
places, like `crypto/x509/v3_addr.c`